### ISSUE TITLE
Can't override default keystroke

### ISSUE THREAD

**Issue by lewisl** [wch/SendText#17]
I follow your instructions to change the key stroke by inserting the following:

```
[
{ "keys": ["super+enter"], "command": "send_selection" }
]
```
into Preference/Sendtext/Key Bindings - User

and editing the file as above.

This is a total NOOP.  Your instructions change the Default key binding file, but that is impossible.  That file is read only.

The package is dead to me.

**Comment by ronnyworm** [comment]
Same thing for me :(
@lewisl did you find an alternative?

**Comment by lewisl** [comment]
No.  I just copy/paste.  Note that nothing happened since 9/21/2018.  Many of these packages are dead and SublimeText HQ refuses to curate the most important packages or simply incorporate the functionality.  The original author does something nice and then gets a day job or otherwise moves on with no future maintainer.  For Sublime Text, this is exacerbating a long slow decline for a wonderful product. But, Will Bond can't be bothered. His and our loss.  Mostly I use Visual Studio Code except for the few things for which Sublime Text remains much better.
